### ISSUE TITLE
Change font to Helvetica (Neue) in Linux

### ISSUE THREAD

**Issue by gozzilli** [matplotlib/matplotlib#2778]
I'm unable to use the fonts Helvetica and Helvetica Neue in Ubuntu Linux.
I receive the error:
```bash
/usr/local/lib/python2.7/dist-packages/matplotlib-1.3.0-py2.7-linux-x86_64.egg/matplotlib/font_manager.py:1236: 
UserWarning: findfont: Font family ['Helvetica Neue'] not found. Falling back to Bitstream Vera Sans
```
Note that:
1. I removed the font cache (but it never got regenerated. Could that be the problem?)
2. The *same* font on MacOS works fine with matplotlib. 
3. I'm happy to attach the font if necessary (and if it's allowed)

matplotlib version: 1.3.0 and master (1.4.x)
ubuntu version: 13.04

I reported the steps I went through for troubleshooting on stackoverflow:
http://stackoverflow.com/questions/21461155/

In short:
```python
import matplotlib as mpl
mpl.use('Agg')
import matplotlib.pyplot as plt
import matplotlib.font_manager as font_manager

path = '/home/<myusername>/.fonts/HelveticaNeue.ttf'

prop = font_manager.FontProperties(fname=path)
prop.set_weight = 'light'
mpl.rcParams['font.family'] = prop.get_name()
mpl.rcParams['font.weight'] = 'light'

fig, ax = plt.subplots()
ax.set_title('Text in a cool font', fontproperties=prop, size=40)
plt.savefig('/tmp/test2.png')
```

**Comment by tacaswell** [comment]
@gozzilli Closing this as it has had no traffic in a year.

I suspect that the issue is related to needing to re-build your font cache.

Does this issue persist on the current release?

Ping to have this re-opened if you can still reproduce this issue.

**Comment by gozzilli** [comment]
@tacaswell Does the font cache matter when you are specifying a font's path directly?
Anyway, the problem does not persist on current release, so all good. Thanks!

**Comment by tacaswell** [comment]
Clearing the cache never hurts!
